samples/csharp_dotnetcore/12.customQABot/README.md

```diff
@@ -0,0 +1,88 @@
+﻿# Custom Question Answering
+
+Bot Framework v4 Custom questino answering bot sample.
+
+This bot has been created using [Bot Framework][BF], it shows how to create a bot that uses the [Custom question answering feature in Language Service][LS] service.
+
+The [Custom question answering feature in Language Service][LS] enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes. In this sample, we demonstrate how to use the Custom question answering feature in Language Service to answer questions based on a FAQ text file used as input.
+
+# Prerequisites
+- Create a [Language resource](https://aka.ms/create-language-resource) with Custom question answering enabled.
+- Follow instructions [here][Quickstart] to create a Custom question answering project. You will need this project's name to be used as `QnAKnowledgebaseId` in [appsettings.json](appsettings.json).
+
+### Obtain values to connect your bot to the knowledge base
+- Follow these steps to update [appsettings.json](appsettings.json).
```

Comment:
No bullet for this..

Follow these steps to update [appsettings.json](appsettings.json).